src/test/java/Homework18.java

```diff
@@ -0,0 +1,77 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+    @Test
+    public void playSong() {
+
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+        navigatetoPage();
+        provideEmail("daria.pavlyuk@testpro.io");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        String songTitleKeyword = "Pluto";
+        searchSong("Pluto");
+        viewAllSearchResults();
+        selectFirstSongResult();
+        setButtonPlaySong();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example

![18](https://github.com/Test-Pro-Bootcamp/A46-QA-Automation-Framework/assets/104761482/aac125cf-911b-49e1-99c4-c7a5fbd1c628)